test/parallel/test-buffer-alloc.js

```diff
@@ -804,19 +804,41 @@ assert.strictEqual(Buffer.from('13.37').length, 5);
 Buffer.from(Buffer.allocUnsafe(0), 0, 0);
 
 // issue GH-5587
-assert.throws(() => Buffer.alloc(8).writeFloatLE(0, 5), RangeError);
-assert.throws(() => Buffer.alloc(16).writeDoubleLE(0, 9), RangeError);
+assert.throws(() => Buffer.alloc(8).writeFloatLE(0, 5), {
+  code: 'ERR_OUT_OF_RANGE',
```

Comment:
The object can be moved to a function, for example:
```
const getRangeError = (min, max, received) => ({
  code: 'ERR_OUT_OF_RANGE',
  name: 'RangeError [ERR_OUT_OF_RANGE]',
  message: `The value of "offset" is out of range. It must be >= ${min} ` +
  'and <= ${max}. Received ${received}'
});
```

And reused across the file